crates/db/src/commands.rs

```diff
@@ -110,6 +110,21 @@ pub async fn db_insert_contract_with_etherscan(
     Ok(())
 }
 
+#[tauri::command]
+pub async fn db_insert_proxy_contract(
```

Comment:
not sure I understand the intent for this command

what I'd imagine is that when adding a contract in the previous methods, we would check ourselves if it's a proxy or not. not relying on the caller to tell us that

this needs to be done in:
- the existing commands for adding contracts
- the anvil expanders that detect on-chain contracts automatically